sdk/core/azure-core/pom.xml

```diff
@@ -145,6 +151,12 @@
       <version>1.22</version> <!-- {x-version-update;org.openjdk.jmh:jmh-generator-annprocess;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-resources</artifactId>
+      <version>2.0.0</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Why do we need resource manager dependency in azure-core?

Reply:
Looks like an accidental dependency added by IntelliJ's add dependency quick fix option. Removed.